src/Evolution/Systems/CurvedScalarWave/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pup.h>
+
+#include "Domain/BoundaryConditions/BoundaryCondition.hpp"
+#include "Domain/BoundaryConditions/Periodic.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace CurvedScalarWave::BoundaryConditions {
+
+template <size_t Dim>
+class Outflow;
+
+}  // namespace CurvedScalarWave::BoundaryConditions
+
+/// \brief Boundary conditions for the curved scalar wave system
```

Comment:
Some older versions of Doxygen don't pick up these nested namespace correctly. I'm very guilty of not having known that early enough and contributed to many of these style comments. Just an FYI, I'm not sure if there was ever a consensus on how to handle this Doxygen version issue

Reply:
I think I will leave this. clang-tidy has a nested namespace warning if you dont use it afaik and as long as spectre-code.org is not parsed by this older version it should not matter too much.

Reply:
Sounds good :) I don't think Doxygen complains if there is any "code" (including comments) between the namespaces, but I agree that as long as the public dox is fine, we're good :)